packages/apsara-ui/src/Listing/Listing.tsx

```diff
@@ -69,11 +65,8 @@ const Listing = ({
             <VirtualisedTable
                 items={filteredList}
                 columnsData={columns}
-                rowKey={rowKey}
                 selectedRowId={selectedRowId}
                 rowClick={rowClick}
-                calculateRowHeight={calculateRowHeight}
-                calculateColumnWidth={calculateColumnWidth}
                 sortable={sortable}
                 {...extraTableProps}
```

Comment:
instead of implicitly assigning the props, let's just assign them manually one by one.

Doing that would help us understand what `VirtualisedTable` props are used by Listing comp.

Reply:
`tableProps` is type of `IVirtualTable` which is further extending `TableProps` (RcTableProps) so it has about 45 props and imo it's not feasible to manually define these many props